eng/Versions.props

```diff
@@ -32,9 +32,9 @@
     <CodeStyleLayerCodeAnalysisVersion>3.6.0-2.final</CodeStyleLayerCodeAnalysisVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20407.3</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.7.0-3.20271.4</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.4.248</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.8.39</VisualStudioEditorPackagesVersion>
```

Comment:
I think because we are updating the VisualStudioEditorPackagesVersion packages to 16.8 this needs to go into master-vs-deps (we don't ship these).

Note that the packages covered by MicrosoftVisualStudioLanguageServerProtocolPackagesVersion can safely use 16.8 versions because we ship the versions we consume.

Reply:
> I think because we are updating the VisualStudioEditorPackagesVersion packages to 16.8 this needs to go into master-vs-deps (we don't ship these).

Sure.  I can rebase this.

Reply:
16,8 p1 is already out, right? So if this is not newer than the version shipped in that, I guess it's OK to target master?

Reply:
I believe the previous standard has been the latest non-preview release

Reply:
Ah, OK. I always get confused on this :)